### ISSUE TITLE
Option to print variables one per line

### ISSUE THREAD

**Issue by pmackay** [gazay/gon#259]
gon output currently has no newlines in it, which is compact but can be hard to read. For usability it would be helpful to have an option that would format the gon <script> content with each variable assignment on separate lines.

**Comment by gazay** [comment]
Hey! Do you mean like pretty-print it in html? When I debug gon output I just use developer console because it provides you beautiful way to interact with json objects (you can just write `window.gon` and after that you can work with whole object using mouse).

**Comment by pmackay** [comment]
>Do you mean like pretty-print it in html?

Yes exactly, so the block of JS inside the `<script>` tag is readable with one variable per line.
